it/controllers/benefits/medical/EducationalServicesAmountControllerISpec.scala

```diff
@@ -0,0 +1,462 @@
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an BAD_REQUEST status" in {
+            result.status shouldBe BAD_REQUEST
+          }
+
+          titleCheck(get.expectedErrorTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(user.commonExpectedResults.expectedCaption)
+          textOnPageCheck(user.commonExpectedResults.previousExpectedContent, contentSelector)
+          textOnPageCheck(user.commonExpectedResults.amountHint, hintTextSelector)
+          textOnPageCheck(poundPrefixText, poundPrefixSelector)
+          inputFieldValueCheck("100,000,000,000", inputSelector)
+
+          buttonCheck(user.commonExpectedResults.continue, continueButtonSelector)
+          formPostLinkCheck(continueLink, continueButtonFormSelector)
+          welshToggleCheck(user.isWelsh)
+
+          errorSummaryCheck(get.maxAmountErrorText, expectedErrorHref)
+          errorAboveElementCheck(get.maxAmountErrorText)
+        }
+      }
+    }
+
+    "redirect to another page when a valid request is made and then" should {
+      "redirect to check employment benefits page when a valid form is submitted and a prior submission" when {
+        implicit lazy val result: WSResponse = {
+          authoriseAgentOrIndividual(isAgent = false)
+          dropEmploymentDB()
+          insertCyaData(employmentUserData(isPrior = true, cyaModel(hmrc = true, Some(benefits(fullMedicalChildcareEducationModel)))), userRequest)
+          urlPost(pageUrl(taxYearEOY), follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)), body = Map("amount" -> "123.45"))
+        }
+
+        "has an SEE_OTHER status" in {
+          result.status shouldBe SEE_OTHER
+
+          result.header("location") shouldBe Some(CheckYourBenefitsController.show(taxYearEOY, employmentId).url)
+        }
+
+        "updates the CYA model with the new value" in {
+          lazy val cyaModel = findCyaData(taxYearEOY, employmentId, userRequest).get
+          val educationalServices = cyaModel.employment.employmentBenefits.flatMap(_.medicalChildcareEducationModel.flatMap(_.educationalServices))
+          educationalServices shouldBe Some(123.45)
+        }
+      }
+
+      "redirect to to Beneficial loans yes/no page when a valid form is submitted and not prior submission" when {
+        implicit lazy val result: WSResponse = {
+          authoriseAgentOrIndividual(isAgent = false)
+          dropEmploymentDB()
+          insertCyaData(employmentUserData(isPrior = false, cyaModel(hmrc = true, Some(benefits(fullMedicalChildcareEducationModel)))), userRequest)
+          urlPost(pageUrl(taxYearEOY), follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)), body = Map("amount" -> "234.56"))
+        }
+
+        "has an SEE_OTHER status" in {
+          result.status shouldBe SEE_OTHER
+          // TODO: Should go to Beneficial loans yes/no page
```

Comment:
again this page has been created if you want to route it there and update the tests